site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Description: This document explains how to setup Prometheus and configure it to scrape data from MinIO servers.
+        SEO Keywords: Prometheus
 
   - Related Projects:
       - Generate Let's Encrypt certificate using Certbot for Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/generate-lets-encypt-certificate-using-certbot-for-minio.md
         Slug: generate-let-s-encypt-certificate-using-concert-for-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Using the Let's Encrypt Certbot with Minio
+        SEO Description: In this recipe, we will generate a Let's Encrypt certificate using Certbot. This certificate will then be deployed for use in the Minio server.
+        SEO Keywords: Let's Encrypt, Certbot, Certificate Authority
 
       - How to install Golang?:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/how-to-install-golang.md
         Slug: how-to-install-golang
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Installing Golang
+        SEO Description: The guide covers the basics of installing Golang
+        SEO Keywords: Golang
 
   - Minio Deployment:
       - Minio Deployment Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/README.md
         Slug: minio-deployment-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | MinIO Deployment Quickstart Guide
+        SEO Description: This guide covers deployment documentation for Orchestration platforms - Docker Swarm, Docker Compose, Kubernetes, DC/OS
+        SEO Keywords: Docker Swarm, Docker Compose, Kubernetes, DC/OS
 
       - Deploy Minio on Docker Swarm:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/docker-swarm/README.md
         Slug: deploy-minio-on-docker-swarm
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Deploy MinIO on Docker Swarm
+        SEO Description: This guide covers how MinIO server can be easily deployed in distributed mode on Swarm to create a multi-tenant, highly-available and scalable object store.
+        SEO Keywords: Docker Swarm, Docker Compose
 
       - Deploy Minio on Kubernetes:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/kubernetes/README.md
         Slug: deploy-minio-on-kubernetes
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Deploy MinIO on Kubernetes
+        SEO Description: This guide documents the multiple options to deploy MinIO on Kubernetes. Concepts like Deployments and StatefulSets MinIO in standalone, distributed or gateway mode.
+        SEO Keywords: kubernetes, statefulsets, helm chart repository
```

Comment:
Once again we can indicate that this is documentation to Deploy Minio on Kubernetes in standalone, distributed and gateway modes